tailwind.config.js

```diff
@@ -3,17 +3,60 @@ const colors = require('@tailwindcss/ui/colors')
 const plugin = require('tailwindcss/plugin')
 
 const colorPrimary = {
```

Comment:
Should we go only with `100, 200, 300, 500, 600` shades? Why not from 100 to 900? 🤔  I have also that same question for `colorSecondary`.

Reply:
I do need to tweak the shades still. The goal was to only have 5-6 shades for primary and secondary since they are a bit brighter and more specific to the brand, whereas for `gray` you want to have at least 10 shades so that you have a wide variety of background colors and text colors. I will probably include `100, 200, 300, 400, 500, 600` shades for both `primary` and `secondary`. Do you think this is okay?

Reply:
The shades for both primary and secondary have been tweaked to include 100 - 600. Let me know if we can resolve this!

Reply:
> Do you think this is okay?

Yeah, it will be :)

 > The shades for both primary and secondary have been tweaked to include 100 - 600. Let me know if we can resolve this!
 
 I will look at this :)

Reply:
Ok, awesome! :)